src/pages/Admin/AdminVenuePreview.tsx

```diff
@@ -30,12 +31,22 @@ export const AdminVenuePreview: React.FC<AdminVenuePreview> = ({
     switch (venue.template) {
       case VenueTemplate.artpiece:
         return (
-          <div>
-            <span className="title">iFrame URL</span>
-            <span className="content">
-              <a href={venue.embedIframeUrl}>{venue.embedIframeUrl}</a>
-            </span>
-          </div>
+          <>
+            <div>
+              <span className="title">iFrame URL</span>
+              <span className="content">
+                <a href={venue.embedIframeUrl}>{venue.embedIframeUrl}</a>
+              </span>
+            </div>
+            <iframe
```

Comment:
Is it worth adding some text above (`Preview your video` or something)?
I'm worried this will look weird, could you upload a screenshot?

Reply:
<img width="1440" alt="Screenshot 2020-08-18 at 11 27 54" src="https://user-images.githubusercontent.com/69144671/90502572-f81f1100-e145-11ea-80d5-3c4c4e4991a8.png">
<img width="1440" alt="Screenshot 2020-08-18 at 11 28 08" src="https://user-images.githubusercontent.com/69144671/90502590-fead8880-e145-11ea-9a80-9eecb87f4e35.png">